src/components/Withdraw/WithdrawPage.tsx

```diff
@@ -155,28 +154,28 @@ export const WithdrawPage = () => {
       }
     };
 
-  if (loadingContracts && walletAvailable) {
+  if (!addressExists) {
     return (
       <div className="flex flex-grow items-center justify-center">
         <Loading />
       </div>
     );
   }
 
-  const isAdding = possibleWithdraws.some((contract) => contract.adding);
+  const isAdding = possibleWithdraws?.some((contract) => contract.adding);
   const contractsToWitdhraw =
-    possibleWithdraws.filter((contract) => contract.added).length === 0;
+    possibleWithdraws?.filter((contract) => contract.added).length === 0;
 
   return (
     <div className="flex h-fit flex-grow flex-col items-center justify-between gap-3 text-m-disabled">
-      {possibleWithdraws.length === 0 ? (
+      {possibleWithdraws?.length === 0 || !addressExists ? (
         <div className="m-auto flex flex-col items-center gap-3 rounded-md p-5 text-center text-3xl font-semibold">
           <Image
             src={NoWithdrawalIcon as string}
             alt=""
             height={ICON_SIZES.XXL}
           />
-          {walletAvailable
+          {addressExists
             ? "No pending withdrawals"
             : "Please connect your wallet to see your pending withdrawals"}
```

Comment:
I still can't see the `Please connect ...` message

Reply:
i think its done :smiling_face_with_tear: